### PR TITLE
Add bazel options

### PR DESCRIPTION
**Pull request by purkhusid**
## What?
Adds options to forward additional bazel startup/command options to the bazel client

# Why?
You might have multiple Bazel invocation in your CI script and you want to use the same flags for all of them so that we can use the same Bazel server for the whole run

### CODE REVIEW

File: src/main/java/com/bazel-diff/main.java [hunk 1]
```diff
@@ -80,13 +80,19 @@ public Integer call() {
     @Option(names = {"-m", "--modifiedFilepaths"}, description = "The path to a file containing the list of modified filepaths in the workspace, you can use the 'modified-filepaths' command to get this list")
     File modifiedFilepaths;
 
+    @Option(names = {"-so", "--bazelStartupOptions"}, description = "Additional Bazel client startup options used when invoking Bazel")
+    String bazelStartupOptions;
+
+    @Option(names = {"-co", "--bazelCommandOptions"}, description = "Additional Bazel command options used when invoking Bazel")
+    String bazelCommandOptions;
+
```
Comment (tinder-maxwellelliott) [inline src/main/java/com/bazel-diff/main.java:88]: These should not be needed, you can refer to these values off of parent like so

`parent.bazelCommandOptions `

Comment (purkhusid) [reply src/main/java/com/bazel-diff/main.java]: @tinder-maxwellelliott Fixed it

**Comment by tinder-maxwellelliott** [comment]
@purkhusid we updated the CLA one last time, once you sign we can merge this! 😄

**Comment by purkhusid** [comment]
@tinder-maxwellelliott Done!

**Pull request merged**